stdlib/in_channel.mli

```diff
@@ -0,0 +1,145 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 2019 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+
+(** Input channels.
+
+    @since 4.10 *)
+
+type t = in_channel
+
+val stdin : t
+(** Standard output. *)
+
+(** {2 General input functions} *)
```

Comment:
Headings start at `1` in modules.

Reply:
Also I think it would be nice to uniformize the docs strings to the format "[func a0 a1] description" of the resulting value or the effect (for `unit`).

Reply:
Fixed, but the docstrings could use some more work I think.